ProjectTemplates/test-templates-from-nuget.bat

```diff
@@ -280,7 +284,7 @@ cd ..
 cd ..
 
 echo "Configure the applications"
-..\..\..\..\tools\ConfigureGeneratedApplications\bin\Debug\net5.0\ConfigureGeneratedApplications.exe
+..\..\..\..\tools\ConfigureGeneratedApplications\bin\Debug\netcoreapp3.1\ConfigureGeneratedApplications.exe
```

Comment:
When testing preview templates the tooling caused issue. Falling back to building it only on .NET Core 3.1 to make things easier